frontend/packages/ceph-storage-plugin/src/components/dashboard-page/storage-dashboard/storage-efficiency-card/storage-efficiency-card-item.tsx

```diff
@@ -0,0 +1,34 @@
+import * as React from 'react';
+import { FieldLevelHelp } from '@console/internal/components/utils';
+
+export const EfficiencyItemBody: React.FC<EfficiencyItemBodyProps> = React.memo(
+  ({ title, stats, infoText, isLoading, error, handler }) => {
+    let status: React.ReactElement;
+    const metricStats = stats ? handler() : null;
```

Comment:
As per the below suggestion:
```suggestion
    const metricStats = metricStatsHandler();
```